src/main/resources/data/trinkets/entities/techreborn.json

```diff
@@ -0,0 +1,8 @@
+{
+	"entities": [
+		"player"
+	],
+	"slots": [
+		"chest/back"
```

Comment:
Is that a usual trinket slot or did you make it up? Because a "/" in the name looks not so good since it is a directory separator

Reply:
It is one of the default trinket slots build-in for the trinkets mod.

If only Tech Reborn and Trinkets are installed, this enables the back trinket when hovering over the chest inventory slot.

Reference:
https://github.com/emilyploszaj/trinkets/wiki/Default-Slots